tcmur_cmd_handler.c

```diff
@@ -2308,6 +2319,320 @@ static int handle_inquiry(struct tcmu_device *dev, struct tcmulib_cmd *cmd)
 	return ret;
```

Comment:
It would be safer to return const uint8_t *, to prevent any callers from accidentally modifying the static data.  It appears to be used safely for now, though.

Reply:
Thanks for your suggestion.
Updated patches.